<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>371</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>371</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>371</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="gridSize">
    <size>
     <width>30</width>
     <height>36</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>General</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/settings/images/AddDownload.png</normaloff>:/settings/images/AddDownload.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Save To</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/settings/images/harddisk.png</normaloff>:/settings/images/harddisk.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Categories</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/settings/images/extensionmanager.png</normaloff>:/settings/images/extensionmanager.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Network</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/settings/images/network-idle.png</normaloff>:/settings/images/network-idle.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="optionStacks">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="GeneralPage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>421</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Connections per download</string>
        </property>
        <property name="buddy">
         <cstring>ConnectionsPerDownloadSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ConnectionsPerDownloadSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="showDownloadsCompleteDialogCheckbox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Download Complete Dialog</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="EnableSoundsCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Sounds</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>421</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max Running Downloads</string>
        </property>
        <property name="buddy">
         <cstring>maxTasksSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxTasksSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="SavetoPage">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>421</width>
       <height>321</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>131</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="title">
      <string>Locations</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>421</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>General</string>
           </property>
           <property name="buddy">
            <cstring>pushButton</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="generalDirectoryEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Compressed</string>
           </property>
           <property name="buddy">
            <cstring>pushButton_6</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="CompressedDirectoryEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Documents</string>
           </property>
           <property name="buddy">
            <cstring>pushButton_2</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="DocumentsDirectoryEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Music</string>
           </property>
           <property name="buddy">
            <cstring>pushButton_3</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="MusicDirectoryEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Programs</string>
           </property>
           <property name="buddy">
            <cstring>pushButton_5</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>31</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="ProgramsDirectoryEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Video</string>
           </property>
           <property name="buddy">
            <cstring>pushButton_4</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="VideoDirectoryEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Categoriespage">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>421</width>
       <height>311</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>131</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="title">
      <string>Categories</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>421</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Compressed</string>
           </property>
           <property name="buddy">
            <cstring>CompressedExtensionsEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="CompressedExtensionsEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Documents</string>
           </property>
           <property name="buddy">
            <cstring>DocumentsExtensionsEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="DocumentsExtensionsEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Music</string>
           </property>
           <property name="buddy">
            <cstring>MusicExtensionsEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="MusicExtensionsEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Programs</string>
           </property>
           <property name="buddy">
            <cstring>ProgramsExtensionsEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>31</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="ProgramsExtensionsEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Video</string>
           </property>
           <property name="buddy">
            <cstring>VideoExtensionsEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="VideoExtensionsEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="NetworkPage">
    <widget class="QGroupBox" name="proxySupport">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>431</width>
       <height>222</height>
      </rect>
     </property>
     <property name="title">
      <string>Enable proxy</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="proxyType">
        <item>
         <property name="text">
          <string>Socks5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Http</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Host:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="proxyHostName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Port:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="proxyPort">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>1080</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>293</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>User Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="proxyUserName"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="proxyPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>maxTasksSpinBox</tabstop>
  <tabstop>ConnectionsPerDownloadSpinBox</tabstop>
  <tabstop>showDownloadsCompleteDialogCheckbox</tabstop>
  <tabstop>EnableSoundsCheckBox</tabstop>
  <tabstop>generalDirectoryEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>CompressedDirectoryEdit</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>DocumentsDirectoryEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>MusicDirectoryEdit</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>ProgramsDirectoryEdit</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>VideoDirectoryEdit</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>CompressedExtensionsEdit</tabstop>
  <tabstop>DocumentsExtensionsEdit</tabstop>
  <tabstop>MusicExtensionsEdit</tabstop>
  <tabstop>ProgramsExtensionsEdit</tabstop>
  <tabstop>VideoExtensionsEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
